ocs_ci/ocs/resources/pod.py

```diff
@@ -370,3 +371,20 @@ def get_osd_pods(osd_label=constants.OSD_APP_LABEL, namespace=None):
     osds = get_pods_having_label(osd_label, namespace)
     osd_pods = [Pod(**osd) for osd in osds]
     return osd_pods
+
+
+def get_resource_obj(name, kind, namespace=None):
```

Comment:
Please change the function name to get_pod_obj

Reply:
done